dev-util/android-tools/metadata.xml

```diff
@@ -6,7 +6,7 @@
 		<changelog>https://android.googlesource.com/platform/packages/modules/adb/+log</changelog>
 		<doc>https://android.googlesource.com/platform/packages/modules/adb/+/HEAD/OVERVIEW.TXT</doc>
 		<remote-id type="cpe">cpe:/a:google:android_debug_bridge</remote-id>
-		<remote-id type="github">android/platform_system_core</remote-id>
+		<remote-id type="github">nmeum/android-tools</remote-id>
 		<remote-id type="launchpad">ubuntu</remote-id>
```

Comment:
By the way, does this line mean anything?

Reply:
Thanks, removed in 79429e7d0a3d11445044344e4e4c67a3e5aee130.